Wrote blog post about Vinny Harris-Riviello and added 3 photos (#2)
msse-2021-bootcamp/team2-blog: _posts/2021-08-09-vinny_harris-riviello.md

---
layout: post
title: "Vinny Harris-Riviello"
author: Kevin Fong
---

<center>

<img src = '{{ "/images/vinny.JPG" | relative_url }}'>   

</center>

# Where is she from? Where does she live?

Vinny grew up in Mexico City for 17 years before she moved to Arizona. She moved around several times in the US before her current residence in Corvalis, Oregon.

# Pets?

Vinny has a strong affinity with animals, but unfortunately she is susceptible to allergies, and thus cannot have any of her own. She loves cat, dogs, but also non-domestic animals, like otters!

# Food Preferences?

> Vinny loves french fries! not just McDonald's french fries, she loves to make her own french fries at home! She experiments with different type of potatoes with different combinations of flavors.  For example she might get a purple magic potatoes, cut them really thin, then season with toasted sesame oil and spices like paprika and garlic powder, with a little bit of pepper. She like to pair with a homemade sauce, like ranch dressing with salsa to give it a nice kick!

Reviewer: ```suggestion
Vinny loves french fries! not fast food type french fries, she loves to make her own french fries at home! She experiments with different type of potatoes with different combinations of flavors.  For example she might get a purple magic potatoes, cut them really thin, then season with toasted sesame oil and spices like paprika and garlic powder, with a little bit of pepper. She like to pair with a homemade sauce, like ranch dressing with salsa to give it a nice kick!
```

She's not vegetarian, but she made a conscious decision to almost completely eliminate meat from her diet for health reasons. That being said, she might have meat a few times a year, because she doesn't want to close off oppurtunities in her life. However she will avoid any offal, or organ meats from animals, at any cost.

# Hobbies: Gardening

> Vinny absolutely love gardening. As she would say, it's great to have more flowers for pollinators, but it's also rewarding to grow herbs and vegatables that she gets to share with family. She's been gardening since 2012, and over the years she's continued to learn about different techniques and plants to try out. She has 2 sizable plots at work where she can do her personal gardening.  For her own challenge, she tries not to grow conventional plants, but oddities like exotic melons or different varieties of strawberries or eggplants. 

Reviewer: ```suggestion
Vinny absolutely loves gardening. As she would say, it's great to have more flowers for pollinators, but it's also rewarding to grow herbs and vegatables that she gets to share with family. She's been gardening since 2012, and over the years she's continued to learn about different techniques and plants to try out. She has 2 sizable plots at work where she can do her personal gardening.  For her own challenge, she tries not to grow conventional plants, but oddities like exotic melons or different varieties of strawberries or eggplants. 
```

<img src = '{{ "/images/IMG-9011.JPG" | relative_url }}'>  
<img src = '{{ "/images/eggplant.JPG" | relative_url }}'> 

# Travelling!

> If Vinny had no responsibilities for a day, she would, without a doubt, go to the beach with some friends. Before school she had a sabbatical year where she traveled the world with my friends. She traveled within the States, but also to Mexico and India. For Vinny, her sabbatical year was a chance to disconnect from the rigor of life to reconnect with the self and spirituality. The chance to return to the beach gives her a chance to reminisce on the simpler days.

Reviewer: ```suggestion
If Vinny had no responsibilities for a day, she would, without a doubt, go to the beach with some friends. Before school she had a sabbatical year where she traveled the world with her friends. She traveled within the States, but also to Mexico and India. For Vinny, her sabbatical year was a chance to disconnect from the rigor of life to reconnect with the self and spirituality. The chance to return to the beach gives her a chance to reminisce on the simpler days.
```
